service/orchestrator/toe.go

```diff
@@ -100,7 +102,8 @@ func (svc *Service) CreateTargetOfEvaluation(ctx context.Context, req *orchestra
 
 	res = req.TargetOfEvaluation
 
-	logging.LogMessage(log, logging.LoglevelInfo, logging.Create, req, req.TargetOfEvaluation.GetCatalogId())
+	params := []string{fmt.Sprintf("and Catalog '%s'", req.TargetOfEvaluation.GetCatalogId())}
```

Comment:
any specific reasons for not inlining this in the request? casting this into a []string and then using `...` seems unnecessary :)

Reply:
Yes, because it becomes hard to read, I find ;) But I'll change it.